metadata.tex

```diff
@@ -105,6 +105,16 @@
 \definecolor{darkcandyapplered}{rgb}{0.64, 0.0, 0.0}
 \definecolor{goldenbrown}{rgb}{0.6, 0.4, 0.08}
 
+\newcommand{\svg}[2][10]{%
+  {\raisebox{-0.15\height}{\includesvg[height=#1px]{\svgs/\detokenize{#2}.svg}}}%
+}%
+\newcommand{\svgunit}[2][10]{%
+  {\raisebox{-0.1\height}{\includesvg[height=#1px]{\svgs/\detokenize{#2}.svg}}}%
+}%
+\newcommand{\svgeven}[2][10]{%
+  \includesvg[height=#1px]{\svgs/\detokenize{#2}.svg}%
+}%
+
```

Comment:
I'm not sure about this. You already neede to pay attention to which glyph you used and decide between two. Now it's three. 

Suggestions
- Modify glyphs so they all look the same while using the same command
- Have only `\svg` command which handles `unit` and `even` glyphs internally

Reply:
We're not modifying the text and adding glyphs any more. If you want to simplify this, you're welcome to do this after this PR. I won't change it now that I'm 80% done. Those glyphs look much better anyway.

Reply:
BTW, all glyphs looking the same is the issue this PR is aiming to fix.